### PR TITLE
chore: removing fast-component-react-msft dependencies

### PR DESCRIPTION
**Pull request by williamw2**
# Description

Removing / replacing some of the dependencies from Site Utilities, Creator and Component Explorer sites. This does not replace all dependencies in Creator or Explorer but does completely remove dependencies from Site Utilities.

Creator Screenshot:
image

Explorer Screenshot:
image

## Motivation & context

Resolves the following:
Fixes: #4262
Fixes: #4264 
Fixes: #4265 
Fixes: #4266 
Fixes: #4268 
Fixes: #4271 

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Review by nicholasrice — APPROVED** [review]

File: sites/fast-component-explorer/app/explorer.tsx [hunk 1]
```diff
@@ -7,17 +7,12 @@ import {
     ViewerCustomAction,
 } from "@microsoft/fast-tooling-react";
 import React from "react";
-import {
-    neutralLayerL1,
-    neutralLayerL2,
-    StandardLuminance,
-} from "@microsoft/fast-components-styles-msft";
+import { StandardLuminance } from "@microsoft/fast-components-styles-msft";
```
Comment (chrisdholt) [inline sites/fast-component-explorer/app/explorer.tsx:10]: This should be available from the `fast-components` package - do we need to keep this import?

File: sites/site-utilities/src/components/logo/index.tsx [hunk 2]
```diff
@@ -17,6 +16,12 @@ const backgroundStyle = {
     alignItems: "center",
     justifyContent: "space-between",
     padding: "8px",
+    backgroundColor: "var(--neutral-fill-rest)",
+};
+const headerStyle = {
+    fontSize: "14px",
+    fontWeight: 600,
+    lineHeight: "20px",
```
Comment (chrisdholt) [inline sites/site-utilities/src/components/logo/index.tsx:24]: Curious if we want to map to font weight and line height variables from the design system here

**Review by chrisdholt — CHANGES_REQUESTED** [review]
One question and one small change to an import that looks like it got missed. Otherwise, LGTM!

File: sites/fast-component-explorer/app/explorer.tsx (hunk 1 above)
Comment (williamw2) [reply sites/fast-component-explorer/app/explorer.tsx:10]: I don't know how that happened. Must have been a bad copy/paste or something. It was supposed to be "@microsoft/fast-components". Thanks for catching that.

File: sites/site-utilities/src/components/logo/index.tsx (hunk 2 above)
Comment (williamw2) [reply sites/site-utilities/src/components/logo/index.tsx]: No problem. Done.

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**